src/Public/OpenApi.ps1

```diff
@@ -1441,15 +2230,17 @@ function Set-PodeOARouteInfo
         $PassThru
     )
 
-    foreach ($r in @($Route)) {
+    foreach ($r in @($Route))
+    {
         $r.OpenApi.Summary = $Summary
         $r.OpenApi.Description = $Description
         $r.OpenApi.OperationId = $OperationId
         $r.OpenApi.Tags = $Tags
-        $r.OpenApi.Deprecated = $Deprecated.IsPresent
+        $r.OpenApi.Deprecated = $Deprecated.IsPresent -and $Deprecated
```

Comment:
Isn't just the first IsPresent check the same? 🤔

Reply:
yes what I meant was $Deprecated.IsPresent -and $Deprecated.ToBool()
I rewrote as 
  if ($Deprecated.IsPresent)
        {
            $r.OpenApi.Deprecated = $Deprecated.ToBool()
        }